modules/virt-about-conditions-and-events.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-diagnosing-datavolumes-using-events-and-conditions.adoc
+
+[id="virt-about-conditions-and-events.adoc_{context}"]
+= About conditions and events
+
+Diagnose DataVolume issues by examining the output of the Conditions and Events sections
```

Comment:
Mark-up `Conditions` and `Events` here since it's specifically talking about the sections in the outtput.

Reply:
Done